repos/system_upgrade/el7toel8/actors/checkdocker/actor.py

```diff
@@ -0,0 +1,32 @@
+from leapp import reporting
+from leapp.actors import Actor
+from leapp.libraries.common.rpms import has_package
+from leapp.models import InstalledRPM
+from leapp.reporting import create_report, Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class CheckDocker(Actor):
+    """
+    Checks if Docker is installed and warns about its deprecation in RHEL8.
+    """
+
+    name = 'check_docker'
+    consumes = (InstalledRPM,)
+    produces = (Report,)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+        if has_package(InstalledRPM, 'docker'):
+            create_report([
+                reporting.Title('Transition from Docker to Podman in RHEL8'),
+                reporting.Summary('Docker has been deprecated in favour of Podman in Red Hat Enterprise Linux 8. This '
+                                  'means that Docker is no longer supported or available on RHEL8.'),
```

Comment:
The warning should mention that why is this important - as docker is being deprecated it will be removed during the upgrade including the current containers.

Reply:
If we have some KB about migrating from docker to podman it would be very nice to provide a link aswell.